index.html

```diff
@@ -4,9 +4,9 @@
   <head>
     <meta charset="UTF-8" />
     <title>Swagger UI</title>
-    <link rel="stylesheet" type="text/css" href="./swagger-ui.css" />
-    <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
-    <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
+    <link rel="stylesheet" type="text/css" href="./swagger/swagger-ui.css" />
```

Comment:
@Jeffzholy remove the `./` from the beginning of href-s, this is going to translated into a url not a file path:
`something.github.io/./swagger/swagger-ui.css` is not going to be resolved